homeassistant/components/totalconnect/__init__.py

```diff
@@ -3,62 +3,64 @@
 import logging
 
 from total_connect_client import TotalConnectClient
-import voluptuous as vol
 
-from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.config_entries import SOURCE_REAUTH, ConfigEntry
 from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
 from homeassistant.core import HomeAssistant
-import homeassistant.helpers.config_validation as cv
 
-from .const import DOMAIN
+from .const import CONF_USERCODES, DOMAIN
 
 _LOGGER = logging.getLogger(__name__)
 
 PLATFORMS = ["alarm_control_panel", "binary_sensor"]
 
-CONFIG_SCHEMA = vol.Schema(
```

Comment:
We usually deprecate removing config yaml support and log a warning for some time when we find config yaml still being used. We could keep the config schema and ignore it in the code and deprecate the top domain key with cv.deprecated. That will emit a warning automatically.

See eg:
https://github.com/home-assistant/core/blob/f78b02b163eb73e3c4ade80d28512136375dc512/homeassistant/components/somfy_mylink/__init__.py#L48-L52

Reply:
Done